src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -258,7 +253,7 @@ Optimize.onPropositionsUpdate(new AdobeCallbackWithError<Map<DecisionScope, Opti
 static func onPropositionsUpdate(perform action: @escaping ([DecisionScope: OptimizeProposition]?) -> Void)
 ```
 
-* _action_ is invoked with propositions dictionary of type `[DecisionScope: OptimizeProposition]`.
+* _action_ is invoked with propositions dictionary of type `[DecisionScope: OptimizeProposition]`. Errors and empty response for a personalization query are **not** passed back in _action_.
```

Comment:
I think this might read better as "Errors and empty responses for personalization queries..."